commands/service_dev.go

```diff
@@ -130,11 +130,20 @@ func (c *serviceDevCmd) runE(cmd *cobra.Command, args []string) error {
 			return nil
 
 		case r := <-listenResultsC:
-			fmt.Printf("Receive result %s %s: %s\n",
-				pretty.Success(r.TaskKey),
-				pretty.Colorize(color.New(color.FgCyan), r.OutputKey),
-				pretty.ColorizeJSON(pretty.FgCyan, nil, false, []byte(r.OutputData)),
-			)
+			errResult := r.GetError()
+			outputResult := r.GetOutput()
+			if errResult != nil {
+				fmt.Printf("Receive execution error on %s task: %s\n",
+					pretty.Fail(r.TaskKey),
+					pretty.Fail(errResult.Message),
+				)
+			} else {
+				fmt.Printf("Receive result %s %s: %s\n",
```

Comment:
Put a success sign and remove colors to keep consistency with others commands printf

Reply:
I think we shouldn't have it here because this is a log stream and there will be too much noise with signs if we put them. What do you think?

Reply:
Ok

Reply:
I agree with hubert, it will look nicer in the console